### PR TITLE
gh-127627: Add `posix._emscripten_debugger` function

### PR DESCRIPTION
**Pull request by hoodmane**
* Issue: gh-127627

### CODE REVIEW

**Comment by hoodmane** [comment]
I guess since it's private it doesn't make it onto the `os` module.

File: Misc/NEWS.d/next/Library/2024-12-05-10-14-52.gh-issue-127627.fgCHOZ.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added ``os._emscripten_debugger()`` to help with debugging the test suite on
```
Comment (mdboom) [inline Misc/NEWS.d/next/Library/2024-12-05-10-14-52.gh-issue-127627.fgCHOZ.rst]:
```suggestion
Added ``posix._emscripten_debugger()`` to help with debugging the test suite on
```

**Review by mdboom — APPROVED** [review]
LGTM.

**Review by freakboy3742 — APPROVED** [review]
👍 Seems like a straightforward addition. I guess it makes sense to keep it private initially; it's easy enough to promote to a public API if it seems worthwhile to do so in future.

**Pull request merged**